CopilotKit/packages/react-core/src/hooks/use-copilot-action.ts

```
@@ -156,192 +149,36 @@ export function useCopilotAction<const T extends Parameter[] | [] = []>(
-        if (isNoArgsRenderWait(renderAndWait)) {
-          return renderAndWait(waitProps as ActionRenderPropsNoArgsWait<T>);
-        } else {
-          return renderAndWait(waitProps as ActionRenderPropsWait<T>);
-        }
-      }
-
-      // Return empty Fragment instead of null
-      return createElement(Fragment);
-    }) as any;
+  if (action.name === "*") {
+    return useRenderToolCall(action as UseRenderToolCallArgs<T>);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)